RecoLocalCalo/HcalRecAlgos/plugins/HcalRecAlgoESProducer.cc

```diff
@@ -94,5 +96,19 @@ HcalRecAlgoESProducer::produce(const HcalSeverityLevelComputerRcd& iRecord)
    return myComputer ;
 }
 
+void HcalRecAlgoESProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+    edm::ParameterSetDescription desc;
+    desc.add<uint32_t>("phase",0);
+    desc.add<std::vector<std::string>>("RecoveredRecHitBits");
+    desc.add<std::vector<std::string>>("DropChannelStatusBits");
+    descriptions.add("hcalRecAlgos",desc);
+
+    edm::ParameterSetDescription desc_sevlvl;
+    desc_sevlvl.add<uint32_t>("Level");
+    desc_sevlvl.add<std::vector<std::string>>("RecHitFlags");
+    desc_sevlvl.add<std::vector<std::string>>("ChannelStatus");
+    desc_sevlvl.addVPSet("SeverityLevels",desc_sevlvl);
+}
```

Comment:
Also, if the PSet elements in the VPSet need to be flexible, you can try to remove the
temp stuff and just do:
```
vpsd1.add<int>("Level", 0);
desc.addVPSet("SeverityLevels", vpsd1);
```
or similar, and expert should know the precise syntax. As far as ConfDB is concerned, we'd need the top-level parameters in th egenerated cfi file, but PSets and VPSets can themselves be empty.